podcasts/AppDelegate.swift

```diff
@@ -326,8 +302,10 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
 
             try FeatureFlag.allCases.forEach { flag in
                 if let remoteKey = flag.remoteKey {
-                    let remoteValue = RemoteConfig.remoteConfig().configValue(forKey: remoteKey).boolValue
-                    try FeatureFlagOverrideStore().override(flag, withValue: remoteValue)
+                    let remoteValue = RemoteConfig.remoteConfig().configValue(forKey: remoteKey)
```

Comment:
I feel like we could probably improve the code above (the one that checks if the value changed and does something about it) but the less we change for now, the better. :)

Reply:
Yep, I think we can address that separately and go ahead and get this in 7.65 so any future flags are covered.